rclcpp/include/rclcpp/parameter_client.hpp

```diff
@@ -136,9 +137,9 @@ class AsyncParametersClient
   std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>
   set_parameters(
     std::vector<rclcpp::parameter::ParameterVariant> parameters,
-    std::function<
-      void(std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)
-    > callback = nullptr)
+    std::function<void(
+      std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)> callback =
+    nullptr)
```

Comment:
That command does not alter the file currently in the repository. Can you please double check that it does for you with the current file in the repo?